.yarnrc.yml

```diff
@@ -0,0 +1 @@
+nodeLinker: node-modules
```

Comment:
Also would it be beneficial to change it to pnp or pnpm for example?

Reply:
We could but there few things still to consider. One major one is local development. Since we do not have node modules anymore, some code editors have issues with resolving the source path. This was with `pnp` however so I would experiment with `pnpm`